android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -107,6 +109,17 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         }
         holder.slotLocation.setText(currentSession.getMicrolocation().getName());
 
+        Track currentSessionTrack = currentSession.getTrack();
+        holder.slotTrack.setText(currentSessionTrack.getName());
```

Comment:
And one quick performance tip. Although this is riddled through the code, avoid allocations in `onBindViewholder` method as it gets called again and again for several times. Set a click listener on the create view holder method which uses a state variable to determine which track it has to go to